latex/Specific Aims.tex

```diff
@@ -124,21 +124,25 @@
 % Aim 4: SAMPL -- Goal is learning from the above (a) where we are now, (b) why we fail/what we're missing, and (c) how to do better
 %I've tried to encapsulate that in these revised aims.
 
-{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+%{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+{\bf \underline{Aim 1.} Spur improvements in modeling protonation states, tautomers, and solvation via generation of new physical property datasets.}\\
 We will develop new solution-phase datasets for druglike small molecules. 
 These data can test critical aspects of small molecule modeling (including accounting for interactions and treatment of protonation/tautomeric state) and improve our ability to predict physical properties relevant to drug discovery in new regions of chemical space. 
 We will initially focus on distribution between organic phases and on pKa's and their modulation by solvent environment, using these data to drive improvements in the modeling of ligand interactions.
 
 
-{\bf \underline{Aim 2.} Measure binding of novel host-guest complexes for introductory ligand binding challenges.}\\
+%{\bf \underline{Aim 2.} Measure binding of novel host-guest complexes for introductory ligand binding challenges.}\\
+{\bf \underline{Aim 2.} Measure high affinity binding of drug-like compounds in simple receptors to challenge quantitative models of binding in systems lacking sampling challenges often observed in protein-ligand binding.}\\
```

Comment:
Better!

Perhaps something like "Measure affinities of druglike compounds to high-affinity supramolecular hosts to isolate challenges to quantitative accuracy from receptor flexibility issues"?